unit-tests/py/rspy/devices.py

```diff
@@ -93,6 +93,76 @@ def enabled( self ):
         return self._removed is False
 
 
+def wait_until_all_ports_disabled( timeout = 5):
+    """
+    waits for all ports to be disabled
+    """
+    for retry in range( timeout ):
+        if len( enabled() ) == 0:
+            break
+        time.sleep( 1 )
+    if len( enabled() ) != 0:
+        log.w( 'Could not disable ports, can\'t infer ports for recovery devices' )
+        return
+
+
+def map_devices_with_unknown_ports( known_ports ):
```

Comment:
I think the `known_ports` can be easily calculated -- I'd rather calculate them here than pass them in.
`known_ports = [device.port for device in _device_by_sn.values() if device.port is not None]`

Reply:
NP